stats/record_test.go

```diff
@@ -93,3 +94,107 @@ func TestRecordWithAttachments(t *testing.T) {
 func cmpExemplar(got, want *metricdata.Exemplar) string {
 	return cmp.Diff(got, want, cmpopts.IgnoreFields(metricdata.Exemplar{}, "Timestamp"), cmpopts.IgnoreUnexported(metricdata.Exemplar{}))
 }
+
+func TestResolveOptions(t *testing.T) {
```

Comment:
please rename the function to TestRecordWithOptions

Reply:
Done.